content/influxdb/v2.2/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,81 @@ menu:
 weight: 101
 ---
 
+## v2.2 [TBD]
+
+### Features
+
+#### Replicate data remotely
+
+- Add the option to [replicate data from InfluxDB OSS to InfluxDB Cloud](/influxdb/v2.2/write-data/replication).
+
+This feature helps you to collect data locally from multiple InfluxDB OSS edge deployments, analyze and aggregate the data, and then send the data to one or more buckets on a remote InfluxDB Cloud instance. If InfluxDB OSS loses connectivity to InfluxDB Cloud, the data is sent when network connectivity is restored.
+
+#### Flux updates
+
+- Update to [Flux v0.150.0](/flux/v0.x/release-notes/#v01500-2022-01-19).
+- Option to log Flux queries that are cancelled because of server shutdown.
+
+#### Build maintenance
+
+- Upgrade `protobuf` library.
+- https://github.com/influxdata/influxdb/commit/5a1e375): Build: allow to build on FreeBSD
+
+#### Task metadata
+
+- (https://github.com/influxdata/influxdb/commit/504f0e4): Passing `type=basic` returns task metadata without query text.
+
+#### Metrics
+
+(https://github.com/influxdata/influxdb/commit/3a81166): Added metrics collection for replications
+(https://github.com/influxdata/influxdb/commit/dece95d): TSM compaction metrics via prometheus
+
+- Add the following metrics when data fails to be added to the replication queue:
+  `PointsFailedToQueue`
+  `BytesFailedToQueue`
+
+- Add the following metrics for writes, to indicate the status:
+	- `pointsWriteRequested`
+	- `pointsWriteOk`
+	- `pointsWriteDropped`
+	- `pointsWriteErr`
+	- `timeout`
+
+- Add disk size metrics per shard:
+  - `total`: number of files per shard
+  - `disk_bytes`: data size in bytes for each shard
+
+- Add cache subsystem metrics:
+  - `inuse_bytes`: Current memory consumption of cache
+  - `diskBytes`: Size of most recent snapshot"
+  - `latest_snapshot`: Unix time of most recent snapshot.
+  - `writes_total`: Counter of all writes to cache
+  - `writes_err`: Counter of failed writes to cache.
+  - `writes_dropped`: Counter of writes (with any dropped points) to cache.
+
+- Add WAL subsystem metrics:
+  - `size`: Gauge of size of WAL in bytes
+  - `writes`: Number of write attempts to the WAL
+  - `writes_err`: Number of failed write attempts to the WAL
+
```

Comment:
does it make sense to add these storage metrics [here](http://localhost:1313/influxdb/v2.1/reference/internals/storage-engine/)?
Or are these only a subset of all storage metrics?